tests/legacy-cli/e2e/tests/build/prerender/discover-routes-ngmodule.ts

```diff
@@ -86,7 +86,7 @@ export default async function () {
   const componentNames: string[] = ['one', 'two-child-one', 'two-child-two'];
 
   for (const componentName of componentNames) {
-    await ng('generate', 'component', componentName, '--project', projectName);
+    await ng('generate', 'component', componentName, '--project', projectName, '--no-standalone');
```

Comment:
Is this really needed? When a component is generated in an ngModule based app, the component is generated in non standalone mode.

https://github.com/angular/angular-cli/blob/3db1d81397b8d0b883ab9f7b9e6a64303f7072d2/packages/schematics/angular/application/index.ts#L192

Reply:
Let's try it! I made this change before I actually updated the schematics, so very possible it wasn't actually needed.